src/v1/imports/carts/bigcommerce.ts

```diff
@@ -1,37 +1,38 @@
-import { xhrResponseSource } from "../../../shared/sources/xhr-response-source";
 import { QueryStringContext } from "../../../shared/types";
-import bigcommerceDataSource from "../../../shared/environment-data-sources/bigcommerce"
+import bigcommerceDataSource from "../../../shared/environment-data-sources/bigcommerce";
 
 const bigcommerceTracker = ({ appId, retailId }: Pick<QueryStringContext, "appId" | "retailId">) => {
-    bigcommerceDataSource({
-        transactionEvent(transactionData) {
-            window.tracker(
-              "addTrans",
-              transactionData.id,
-              retailId ?? appId,
-              transactionData.total,
-              transactionData.tax,
-              transactionData.shipping,
-              transactionData.city,
-              transactionData.state,
-              transactionData.country
-            );
-      
-            transactionData.items.forEach((item) => {
-              window.tracker(
-                "addItem",
-                transactionData.id,
-                item.sku,
-                item.name,
-                item.category,
-                item.unitPrice,
-                item.quantity,
-                transactionData.currency
-              );
-            });
-            window.tracker("trackTrans");
-          },
-    });
+  bigcommerceDataSource({
+    transactionEvent(transactionData) {
+      window.tracker(
+        "addTrans",
+        transactionData.id,
+        retailId ?? appId,
+        transactionData.total,
+        transactionData.tax,
+        transactionData.shipping,
+        transactionData.city,
+        transactionData.state,
+        transactionData.country,
+        transactionData.currency
+      );
+
+      transactionData.items.forEach((item) => {
+        window.tracker(
+          "addItem",
+          transactionData.id,
+          item.productId,
+          item.sku,
+          item.name,
+          item.category,
+          item.unitPrice,
+          item.quantity,
+          transactionData.currency
+        );
+      });
```

Comment:
This will not work since it does not adhere to the `addItem` schema. This will fail enrichment and this data will not make it into snowflake.

![image](https://github.com/MediaJel/mediajel-tracker/assets/69985528/5ef38e35-8c23-4878-aa68-cb03cc69f35c)

Reference: https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/